Sources/Sentry/SentryTracer.m

```diff
@@ -649,9 +648,8 @@ - (nullable SentryAppStartMeasurement *)getAppStartMeasurement
 
     NSTimeInterval difference = [appStartEndTimestamp timeIntervalSinceDate:self.startTimestamp];
 
-    // If the difference between the end of the app start and the beginning of the current
-    // transaction is smaller than SENTRY_APP_START_MEASUREMENT_DIFFERENCE. With this we
-    // avoid messing up transactions too much.
+    // Don't attach app start measurements if too much time elapsed between the end of the app start
+    // sequence and the start of the transaction. This makes transactions too long.
```

Comment:
This comment didn't make sense so I reworded it and took a guess as to what "messing up transactions too much" meant. But, I'd like more context around _why_ we don't want to attach such spans.